experimental/auth-logic/wrappers/rekor_wrapper.go

```diff
@@ -122,18 +152,107 @@ func verifyRekordLogSignature(rekordEntry *rekord.V001Entry) (*ecdsa.PublicKey,
+	logEntryPrincipal := fmt.Sprintf(`"%s::RekorLogEntry"`, SanitizeName(endorsementAppName))
 
-	ok = ecdsa.VerifyASN1(ecdsaKey, data, sig)
-	if !ok {
-		return nil, fmt.Errorf("could not verify ecdsa signature. key:%v, data:%v, sig:%v ", ecdsaKey, data, sig)
+	// Unpack rekord log entry from bytes into go structs
+	logEntryAnon, err := getLogEntryAnonFromBytes(rlw.rekorLogEntryBytes)
+	if err != nil {
+		return UnattributedStatement{}, fmt.Errorf("couldn't parse rekor log entry from bytes: %v, %v", rlw.rekorLogEntryBytes, err)
+	}
+	entryImpl, err := getEntryImplFromAnon(*logEntryAnon)
+	if err != nil {
+		return UnattributedStatement{}, fmt.Errorf("couldn't get entryImpl from body of logEntryAnon %v: %v", *logEntryAnon, err)
 	}
 
-	return ecdsaKey, nil
+	rekordEntry, err := getRekordEntryFromEntryImpl(*entryImpl)
+	if err != nil {
+		return UnattributedStatement{}, fmt.Errorf("couldn't get rekordEntry from entryImpl:, %v %v", *entryImpl, err)
+	}
+
+	// Authorization logic statmenets are defined next to the go code that
+	// they correspond to so that this is easier to follow along
+
+	// Verify rekor log entry signature
+	_, err = verifyRekordLogSignature(rekordEntry)
+	if err != nil {
+		return UnattributedStatement{}, fmt.Errorf("couldn't validate signature in rekor log entry %v", err)
+	}
+	logEntrySignatureStatement := fmt.Sprintf("hasValidBodySignature(%v).", logEntryPrincipal)
+
+	// Verify inclusion proof
+	err = checkInclusionProof(logEntryAnon, strfmt.Default)
+	if err != nil {
+		return UnattributedStatement{}, fmt.Errorf("couldn't validate logEntryAnon (which includes inclusion proof checking):%v ", err)
+	}
+	inclusionProofStatement := fmt.Sprintf("hasValidInclusionProof(%v).", logEntryPrincipal)
+
+	// Check that the product team public key in the log entry matches
+	// the input public key
+	err = checkEntryPubKeyMatchesExpectedKey(rekordEntry, rlw.productTeamKeyBytes)
+	if err != nil {
+		return UnattributedStatement{}, fmt.Errorf("rekord entry key does not match input product team key: %v, %v, %v", rekordEntry, rlw.productTeamKeyBytes, err)
+	}
+	pubKeyMatchStatement := fmt.Sprintf("hasCorrectPubkey(%v).", logEntryPrincipal)
+
+	//TODO(#76): check that hash of endorsement file matches hash
+	//of rekor log entry contents. To be able to test this, we need a new
+	//rekor log entry that matches the endorsement file.
+	contentsMatchStatement := fmt.Sprintf("contentsMatch(%v, %v).", logEntryPrincipal, endorsementPrincipal)
+
+	// This is the policy for claiming an endorsement is a valid rekor log entry
+	// which just collects the evidence above into the more compact statement
+	// that the verifier wrapper uses.
+	rekorEntryPolicy := fmt.Sprintf("%v canActAs ValidRekorEntry :- hasValidBodySignature(%v), hasValidInclusionProof(%v), hasCorrectPubKey(%v), contentsMatch(%v, %v).", endorsementPrincipal, logEntryPrincipal, logEntryPrincipal, logEntryPrincipal, logEntryPrincipal, endorsementPrincipal)
```

Comment:
nit: For readability, I'd suggest breaking it into multiple lines, with each argument in a separate line.